recipes/ignition-tools/all/conanfile.py

```diff
@@ -93,13 +97,49 @@ def package(self):
         for dll_pattern_to_remove in ["concrt*.dll", "msvcp*.dll", "vcruntime*.dll"]:
             tools.remove_files_by_mask(os.path.join(self.package_folder, "bin"), dll_pattern_to_remove)
 
+        self._create_cmake_module_variables(
+            os.path.join(self.package_folder, self._module_file_rel_path),
+                tools.Version(self.version))
+
+    @staticmethod
+    def _create_cmake_module_variables(module_file, version):
+        content = textwrap.dedent("""\
+            set(IGN_TOOLS_CONAN_PACKAGE_DIR ${{CMAKE_CURRENT_LIST_DIR/../..}})
+            list(APPEND IGNITION-TOOLS_BINARY_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/bin)
+            list(APPEND IGNITION-TOOLS_INCLUDE_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/include)
+            list(APPEND IGNITION-TOOLS_LIBRARY_DIRS ${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/libs)
+            list(APPEND IGNITION-TOOLS_CFLAGS -I/${{IGN_TOOLS_CONAN_PACKAGE_DIR}}/include)
+            list(APPEND IGNITION-TOOLS_CXX_FLAGS -std=c++11)
+            set(ignition-tools{major}_VERSION_MAJOR {major})
+            set(ignition-tools{major}_VERSION_MINOR {minor})
+            set(ignition-tools{major}_VERSION_PATCH {patch})
+            set(ignition-tools{major}_VERSION_STRING "{major}.{minor}.{patch}")
+        """.format(major=version.major, minor=version.minor, patch=version.patch))
+        tools.save(module_file, content)
+
     def package_info(self):
-        version_major = tools.Version(self.version).major
-        self.cpp_info.names["cmake_find_package"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.names["cmake_find_package_multi"] = "ignition-tools{}".format(version_major)
-
-        self.cpp_info.components["libignition-tools"].libs = ["ignition-tools-backward"]
-        self.cpp_info.components["libignition-tools"].includedirs.append("include/ignition/tools{}".format(version_major))
-        self.cpp_info.components["libignition-tools"].names["cmake_find_package"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.components["libignition-tools"].names["cmake_find_package_multi"] = "ignition-tools{}".format(version_major)
-        self.cpp_info.components["libignition-tools"].names["pkg_config"] = "ignition-tools{}".format(version_major)
+        lib_name = "ignition-tools"
+        self.cpp_info.names["cmake_find_package"] = lib_name
+        self.cpp_info.names["cmake_find_package_multi"] = lib_name
+        self.cpp_info.set_property("cmake_file_name", "ignition-tools")
+        self.cpp_info.components["backward"].names["cmake_find_package"] = "backward"
```

Comment:
I am not sure whats the best option here but the targets are not *correctly* exporting from the upstream project... Does this match the projects intentions? I am not sure I can answer that...

https://cmake.org/cmake/help/latest/guide/importing-exporting/index.html#exporting-targets calls out `install(EXPORT...) and `install(TARGETS ...)` neither exist 😕 

https://github.com/gazebosim/gz-tools/blob/ignition-tools_1.4.0/src/CMakeLists.txt#L29 main gets closer but it's still not doing something consumers would rely on AFAIK.